public/app/features/dashboard/dashgrid/PanelHeader/PanelHeader.tsx

```diff
@@ -39,10 +40,11 @@ export class PanelHeader extends PureComponent<Props> {
             </span>
 
             <PanelHeaderMenu panel={panel} dashboard={dashboard} />
-
-            <span className="panel-time-info">
-              <i className="fa fa-clock-o" /> 4m
-            </span>
+            {timeInfo ? (
+              <span className="panel-time-info">
+                <i className="fa fa-clock-o" /> {timeInfo}
+              </span>
+            ) : null}
```

Comment:
Instead of returning `null`. We can do:
```
timeInfo && (
    <span className="panel-time-info">
        <i className="fa fa-clock-o" /> {timeInfo}
    </span>
)
```

Reply:
Excellent suggestion!